modules/installation-special-config-encrypt-disk-tpm2.adoc

```diff
@@ -4,33 +4,37 @@
 
 [id="installation-special-config-encrypt-disk-tpm2_{context}"]
 = Enabling TPM v2 disk encryption
-Use this procedure to enable TPM v2 mode disk encryption during {product-title} deployment.
+Use this procedure to enable TPM v2 mode disk encryption during an {product-title} installation.
+
+.Prerequisites
+
+* You downloaded the {product-title} installation program on your installation node.
```

Comment:
```suggestion
* You have downloaded the {product-title} installation program on your installation node.
```
For consistency sake with the bullet below.

Reply:
Resolved.